### DISCUSSION TITLE
zotfile功能的替代

### DISCUSSION THREAD

**Discussion by gaochuanchao** [MuiseDestiny/zotero-style#489] [Ideas]
zotfile似乎不支持zotero 7了, 其开发人员也明确说明没有更新计划. 不知道大佬能否出一个相关功能的平替呢?
我个人会比较常用的功能就是 "rename and move", 就是找到新论文时可以根据自己定义的方式对文件重命名并且移动文件到相应的文件夹.

**Comment by MuiseDestiny** [comment]
https://github.com/MuiseDestiny/zotero-attanger
